includes/payment-methods/class-wc-stripe-express-checkout-helper.php

```diff
@@ -699,6 +699,10 @@ private function is_product_or_cart_taxable() {
 
 		// Cart or checkout page: the cart is taxable if any item in the cart
 		// is taxable.
+		if ( ! isset( WC()->cart ) || empty( WC()->cart ) ) {
```

Comment:
We can just use `empty` because it accounts for undefined values.

Reply:
Thanks @wjrosa 
Updated in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4199/commits/a31bd7c856ae52b5fe39fd28f4acd23e295bde72